maquetado/scss/tech.scss

```diff
@@ -0,0 +1,53 @@
+@import 'colors';
+@import 'styles';
+$tech-logo-size: 100px;
+
+.tech-container {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  -webkit-flex-flow: row wrap;
+}
+
+.tech {
+  background: $green;
+  border-radius: 5px;
+  font-size: $text-medium;
+  height: 300px;
+  left: 10px;
+  position: relative;
+  text-align: center;
+  width: 200px;
+
+  &::before {
+    border: 2px solid $green;
+    border-radius: 5px;
+    content: '';
+    height: 300px;
```

Comment:
use sass variables for the height and the width to make it simple to change the dimensions of both elements